QuadradiusR/scripts/embedded_server.gd

```diff
@@ -0,0 +1,87 @@
+extends Node
+
+var SERVER_PORT_FILE = '.server_port'
+var EXECUTABLE_PATH = './quadradiusr_server'
+
+var executable_path: String
+var available: bool
+var pid: int = -1
+var wait_left: float = 0
+var server_port: int = -1
+var on_server_ready_callbacks: Array = []
+
+func _init():
+	self.available = File.new().file_exists(EXECUTABLE_PATH)
+
+	if self.available:
+		print('Server executable is available')
+	else:
+		print('Server executable is not available')
+
+
+func _ready():
```

Comment:
- [ ] ```_ready()``` should be the first function declared, check [here](https://github.com/Fruktus/QuadradiusR/blob/master/CONTRIBUTING.md)

Reply:
I believe `_init` is called earlier than `_ready`